### PR TITLE
fix: Remove quotes from delete dialog #10008

### LINKED ISSUE #10008: Remove quotes from delete dialog
# Summary

Quotation marks should be used to delineate things and make it clear that things are not prose.

There are other ways to delineate such things, e.g. `<tt>text</tt>` instead of `'text'`.

# Motivation

I've watched people try to select the name of the resource in order to delete an object, e.g.:
image

They generally trip on the `'`s (there are magical ways to use a mouse to avoid grabbing them, but, forcing users to deal w/ this isn't very nice).

# Proposal

1. Switch from `'%s'` to `<tt>%s</tt>` (or the equivalent of `<tt>`...)
2. Don't include the name of the resource in the delete line, watching people try to copy the name from the field and having it run away is watching them be tortured...

### PR DESCRIPTION
**Pull request by JessieTeng89**
fix: Remove quotes from delete dialog #10008
Fixes #10008

### CODE REVIEW

File: ui/src/app/applications/components/utils.tsx [hunk 1]
```diff
@@ -330,11 +334,11 @@ export const deletePopup = async (ctx: ContextApis, resource: ResourceTreeNode,
         api => (
             <div>
                 <p>
-                    Are you sure you want to delete {resource.kind} '{resource.name}'?
+                    Are you sure you want to delete {resource.kind} <kbd>{resource.name}</kbd>?
                 </p>
                 {isManaged ? (
                     <div className='argo-form-row'>
-                        <FormField label={`Please type '${resource.name}' to confirm the deletion of the resource`} formApi={api} field='resourceName' component={Text} />
+                        <FormField label={`Please type <kbd>${resource.name}</kbd> to confirm the deletion of the resource`} formApi={api} field='resourceName' component={Text} />
```
Comment (ashutosh16) [inline ui/src/app/applications/components/utils.tsx:341]: The HTML tag should not be displayed in the dialog box, not sure if this is intentional
Screen Shot 2022-08-29 at 12 06 19 PM

**Comment by jsoref** [comment]
Could you provide a picture of the dialog?

**Comment by JessieTeng89** [comment]
> Could you provide a picture of the dialog?

Please find the screenshots for the current changes:
image
image
image

**Review by jsoref — APPROVED** [review]

**Comment by JessieTeng89** [comment]
@rbreeze Can I request your approval? So that we can merge it, or I appreciate with your any comments

**Review by rbreeze — CHANGES_REQUESTED** [review]
`<kbd>` is intended for keyboard input, could we use `<pre>` instead?

**Comment by JessieTeng89** [comment]
> `<kbd>` is intended for keyboard input, could we use `<pre>` instead?

I tried to use `<pre>` instead, we need to add a css style like `display: inline-block` as this tag is a block-level element.

And you can see the change and the style in following screenshot, if it is OK for you, I can do the change
image

**Review by rbreeze — APPROVED** [review]
let’s stick with `kbd`

**Comment by JessieTeng89** [comment]
> let’s stick with `kbd`

Hi @jsoref @rbreeze Can I know how to merge it, it seems I cannot do that even its status is ready

**Comment by jsoref** [comment]
Someone with the ability to merge will manage that (it isn't me and it isn't you).

**Pull request merged**